runner/conmon/pipes.go

```diff
@@ -9,10 +9,24 @@ import (
 	"strings"
 	"time"
 
-	"github.com/containers/podman/v4/libpod/define"
 	"github.com/pkg/errors"
 )
 
+// These errors are adapted from github.com/containers/podman:libpod/define
+// And copied to reduce the vendor surface area of this library
+var (
+	// ErrInternal indicates an internal library error
+	ErrInternal = fmt.Errorf("internal libpod error")
```

Comment:
Would it make sense to rename the error message to:

```suggestion
	ErrInternal = fmt.Errorf("internal error")
```